xarray/tests/test_plot.py

```diff
@@ -1828,6 +1830,106 @@ def test_datetime_line_plot(self):
         self.darray.plot.line()
 
 
+@requires_nc_time_axis
+@requires_cftime
+class TestCFDatetimePlot(PlotTestCase):
+    @pytest.fixture(autouse=True)
+    def setUp(self):
+        '''
+        Create a DataArray with a time-axis that contains cftime.datetime
+        objects.
+        '''
+        month = np.arange(1, 13, 1)
+        data = np.sin(2 * np.pi * month / 12.0)
+        darray = DataArray(data, dims=['time'])
+        darray.coords['time'] = xr.cftime_range(start='2017',
+                                                periods=12,
+                                                freq='1M',
+                                                calendar='noleap')
+
+        self.darray = darray
+
+    def test_cfdatetime_line_plot(self):
+        # test if line plot raises no Exception
+        self.darray.plot.line()
+
+
+@requires_nc_time_axis
+@requires_cftime
+class TestCFDatetimePcolormesh(PlotTestCase):
+    @pytest.fixture(autouse=True)
+    def setUp(self):
+        '''
+        Create a DataArray with a time-axis that contains cftime.datetime
+        objects.
+        '''
+        month = np.arange(1, 13, 1)
+        data = np.random.rand(4, 12)
+
+        darray = DataArray(data, dims=['x', 'time'])
+        darray.coords['time'] = xr.cftime_range(start='2017',
+                                                periods=12,
+                                                freq='1M',
+                                                calendar='noleap')
+
+        self.darray = darray
+
+    def test_cfdatetime_line_plot(self):
```

Comment:
Can you rename this to `test_cfdatetime_pcolormesh_plot` and move it to the `TestCFDatetimePlot` class above?